### DISCUSSION TITLE
How to install pre-releases

### DISCUSSION THREAD

**Discussion by transcental** [python-poetry/poetry#5064] [Q&A] [answered]
Hi!

I'm developing a Python package that provides easy access to GitHub's GraphQL API _(DillonB07/GitAPI#10)_.

In my README, I've got instructions on how to install the latest version of the module. As I understand it you can do this with poetry:

```zsh
poetry add pygitapi@latest
```
But how do you get pre-releases to be installed? Is there a flag like `--pre`?

Thanks,
DillonB07

**Comment by clintonroy** [comment, marked as answer]
I would like to know how you didn't find this in the help.

poetry add --help
  --allow-prereleases    Accept prereleases.

**Comment by transcental** [comment]
Thanks @clintonroy!

I don’t use Poetry myself. I find it way too unnecessarily complicated and I find it easier to just use Pip and virtualenvs.

I looked through the commands on the website and couldn’t find the pre-release option.

On 19 Jan 2022, at 05:27, Clinton Roy ***@***.******@***.***>> wrote:


CAUTION: This email originated from outside of the School. Do not click links or open attachments unless you recognize the sender and know the content is safe. Please contact IT Support if you are unsure.


I would like to know how you didn't find this in the help.

poetry add --help
--allow-prereleases Accept prereleases.

—
Reply to this email directly, view it on GitHub<https://github.com/python-poetry/poetry/discussions/5064#discussioncomment-1995122>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AUAPGD57ZWDJSOSCVQL7YQ3UWZDSNANCNFSM5MH6M35A>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

Note: This message is for the named person's use only. It may contain confidential, proprietary or legally privileged information. If you receive this message in error, please immediately delete it and all copies of it from your system, destroy any hard copies of it and notify the sender. You must not, directly or indirectly, use, disclose, distribute, print, or copy any part of this message if you are not the intended recipient. Beaumont Leys School reserves the right to monitor all e-mail communications through its networks. Any views expressed in this message are those of the individual sender, except where the message states otherwise and the sender is authorized to state them to be the views of any such entity. Thank You.
